src/Accelerators/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+
+if (ACCELERATORS_TO_BUILD)
+  set (ACCEL_LIST "") 
+  foreach(t ${ACCELERATORS_TO_BUILD})
+    message(DEBUG "Targeting ${t}")
+    add_subdirectory(${t})
+    list(APPEND ACCEL_LIST ${t}"Accel")
+  endforeach(t)
+else (ACCELERATORS_TO_BUILD)
+  set (ACCEL_LIST "NoAccel")    
+endif (ACCELERATORS_TO_BUILD)
+
+add_onnx_mlir_library(NoAccel
```

Comment:
By default this function is going to add it to the OMLibs property. Do we want the accelerator libs linked in everywhere the OMLibs are?

Reply:
something has to be provided. That's what the NoAccelerator thing was ... I removed it and just use the Accelerator.cpp to make a lib.

Reply:
Just to make sure I understand correctly, the accelerators will need to be added every place where the OMLibs are used for linking. So in essence, the accelerators will be needed everywhere?

If so, this is ok, otherwise, we should make sure that they don't get added to the OMLibs property.